apps/landingPage/src/components/withSizeDetect.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react';
+
+export default function WithSizeDetect(WrappedComponent) {
```

Comment:
maybe just `withWindowSize` is more descriptive.

Reply:
Renamed